server/odc-server/src/main/resources/config/application.yml

```diff
@@ -75,8 +75,14 @@ management:
     web:
       exposure:
         include: "business,application"
+    enabled-by-default: ${odc.system.monitor.actuator.enabled:false}
+  metrics:
+    export:
+      prometheus:
+        enabled: ${odc.system.monitor.actuator.enabled:false}
+
   endpoint:
     prometheus:
-      enabled: true
+      enabled: ${odc.system.monitor.actuator.enabled:false}
   server:
-    port: 8089
+    port: ${odc.system.monitor.actuator.port:-1}
```

Comment:
Is -1 a illegal port value?

Reply:
Illegal port value, removed default value. Insert value if need to